addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -36,9 +38,16 @@
                                 <small class="ml-2 text-white-50"><t t-esc="slide.channel_id.completion"/> %</small>
                             </div>
                         </div>
+                        <div class="d-block d-sm-none w-100"/>
```

Comment:
Not sure what this empty block is about?

Reply:
Also, not sure how this change is related to the header color?

Reply:
Ok I get the "col-12 col-sm-9 / col-sm-3" part.

But I still don"t get this empty block thing :smile: 

When I play around in debug mode, it doesn't seem to do anyhting.
I can remove it and nothing seems to change.